src/config/section/image.js

```diff
@@ -50,7 +50,11 @@ export default {
           icon: 'plus',
           label: 'Create template',
           listView: true,
-          args: ['url', 'name', 'displaytext', 'directdownload', 'zoneids', 'hypervisor', 'format', 'ostypeid', 'checksum', 'isextractable', 'passwordenabled', 'sshkeyenabled', 'isdynamicallyscalable', 'ispublic', 'isfeatured', 'isrouting', 'requireshvm']
+          popup: true,
+          show: (record, user, apis) => {
+            return Object.prototype.hasOwnProperty.call(apis, 'registerTemplate')
```

Comment:
This is not needed for actions that are rendered in the top bar near breadcrumbs because the actionbutton.vue already has: `v-if="action.api in $store.getters.apis ...`